src/layman/layer/prime_db_schema/bbox_tasks.py

```diff
@@ -0,0 +1,33 @@
+from celery.utils.log import get_task_logger
```

Comment:
I think that added test into this commit is the easiest way. And cancel the second pull request.